guiEditor/src/diagram/workbench.tsx

```diff
@@ -464,7 +503,17 @@ export class WorkbenchComponent extends React.Component<IWorkbenchComponentProps
         const onPointerDown = guiControl.onPointerDownObservable.add((evt) => {
             if (!this.isUp || evt.buttonIndex > 0) return;
             if (this._forceSelecting) {
-                this.isSelected(true, guiControl);
+                // if this is our first click or the clicked control is a parent of the previous clicked control.

```

Comment:
is a parent should be is a child ?

Reply:
also you put or in the sentence but && in code and I can t see which one is right ?